packages/components/inputs/SelectInput.tsx

```diff
@@ -37,34 +45,84 @@ type Props = {
   data: SelectInputData[];
   placeHolder?: string;
   selectedData: SelectInputData;
-  setData: (data: SelectInputData) => void;
+  selectData: (data: SelectInputData) => void;
```

Comment:
name is weird
```suggestion
  onSelect: (data: SelectInputData) => void;
```
also would be better to use a generic type for data value https://www.typescriptlang.org/docs/handbook/2/generics.html
```suggestion
  selectData: (data: SelectInputData<T>) => void;
```

Reply:
type simplified (only string)